interface/weno/transmitDataClass.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * weno rx mark tx.
```

Comment:
I don't understand this one to fix the comment. It is as the other comments. The only thing I can think is that Caps are being requested?